ios/Runner/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>BGTaskSchedulerPermittedIdentifiers</key>
+	<array>
+		<string>dev.flutter.background.refresh</string>
```

Comment:
Maybe also make our own custom task identifier here because apple is notoriously picky for background things. After 15 minutes I hit a hiccup on the latest version of iOS 17 simulator.

Reply:
Looking at this further I think I should remove it from the plist and just use https://developer.apple.com/documentation/uikit/app_and_environment/scenes/preparing_your_ui_to_run_in_the_background/using_background_tasks_to_update_your_app

Reply:
I don't think this entry is necessary, we don't need to update the app in the background only run code in the isolate.

Reply:
Don't I still have to add it for iOS https://pub.dev/documentation/flutter_background_service/latest/ the example of the plist is just for iOS 13+ though. Adding it I thought would resolve the background task and the other issue of it being killed in the background.

Reply:
@kevmo314 ok I updated it opting to use the AppDelegate.swift instead of the plist